### PR TITLE
Use extrenal ci-helpers

### PR DESCRIPTION
**Pull request by bsipocz**
Somehow travis doesn't pick up the branch on my fork, thus had to open this PR here rather than wait while it pass for sure.

### CODE REVIEW

**Comment by bsipocz** [comment]
Well, this is unexpected.
 requests got installed https://travis-ci.org/astropy/astroquery/jobs/90554474#L343 still in fails to import it here https://travis-ci.org/astropy/astroquery/jobs/90554474#L996

cc @astrofrog

**Comment by astrofrog** [comment]
@bsipocz - ah, looks like you figured it out (the latest issue i think is just related to installing the keyring package on Python 3.3?).

@keflavich - just for info, this is a PR for an experimental way of centralizing a lot of the CI stuff that @bsipocz and I are trying out. We'll send more info to the list later.

**Comment by astrofrog** [comment]
@bsipocz - I think ``keyring`` should be pip installed too to avoid the following conflicts:

```
Hint: the following packages conflict with each other:
  - python 3.4*
  - keyring
```

It was pip installed before I think, not conda installed.

**Comment by bsipocz** [comment]
Thanks @astrofrog, I'll check on this. 
Python 3.4 was not in the matrix, and indeed it was pip installed in 3.3.

And sorry @keflavich, I should have said that in the comments, that this is purely experimental at the moment, but thought that it would make some sense to check it out on more packages before sending out the e-mail.

File: .travis.yml [hunk 1]
```diff
@@ -44,16 +47,18 @@ matrix:
...
+        - python: 3.4
           env: ASTROPY_VERSION=development SETUP_CMD='test'
 
-        # Try all python versions with the latest numpy
+        # Try with optional dependencies disabled
         - python: 2.7
           env: SETUP_CMD='test'
-        - python: 3.3
-          env: SETUP_CMD='test'
+               CONDA_DEPENDENCIES='requests'
```
Comment (astrofrog) [inline .travis.yml]: It looks like this won't work without beautiful-soup?

Comment (bsipocz) [inline .travis.yml]: yes, I plan to go through the dependencies one by one to see which are the mandatory build-time ones.

btw do you think it's a problem for core if astropy is pip installed at the time wcsaxes is installed?

Comment (astrofrog) [inline .travis.yml]: @bsipocz - we don't currently install WCSAxes as a dependency for the core on Travis, right? Do you think we should?

Comment (astrofrog) [inline .travis.yml]: (we install it for readthedocs, but not for Travis)

**Pull request merged**